lib/spree_virtual_gift_card/factories.rb

```diff
@@ -12,6 +12,11 @@
 
     factory :redeemable_virtual_gift_card do
       association :purchaser, factory: :user
+      inventory_unit do |gift_card|
+        gift_card.line_item.inventory_units.first ||
```

Comment:
Thats a great point. I updated the factory a little bit to better handle double-dipping on inventory units. Let me know what you think